test/external-e2e/run.sh

```diff
@@ -29,7 +29,7 @@ setup_e2e_binaries() {
     curl -sL https://storage.googleapis.com/kubernetes-release/release/v1.24.0/kubernetes-test-linux-amd64.tar.gz --output e2e-tests.tar.gz
     tar -xvf e2e-tests.tar.gz && rm e2e-tests.tar.gz
 
-    export EXTRA_HELM_OPTIONS="--set driver.name=$DRIVER.csi.k8s.io --set controller.name=csi-$DRIVER-controller --set node.name=csi-$DRIVER-node --set feature.enableInlineVolume=true --set controller.runOnMaster=true"
+    export EXTRA_HELM_OPTIONS="--set driver.name=$DRIVER.csi.k8s.io --set controller.name=csi-$DRIVER-controller --set node.name=csi-$DRIVER-node --set feature.enableInlineVolume=true"
```

Comment:
@andyzhangx considering 1.24 also has label `control-plane` ( for new clusters) , isnt it better to turn runOnControlplane to also `true` ?

Reply:
that would require external e2e test running on 1.24, otherwise it would fail, it's better not set that restriction since controller could also run on agent node nowadays.